controls/def.cf

```diff
@@ -349,7 +349,8 @@ bundle common inventory_control
 # are disabled.
 {
   vars:
-      "lldpctl_exec" string => "/usr/bin/lldpctl";
```

Comment:
For others like `lsb_exec` we put the executables inside `inventory_control` so users have just one place to change.

I'll add the `def` override.